src/olympia/activity/tests/test_utils.py

```diff
@@ -516,7 +516,7 @@ def test_send_activity_mail():
     assert len(mail.outbox) == 1
     assert mail.outbox[0].body == message
     assert mail.outbox[0].subject == subject
-    reference_header = '{addon}/{version}@{site}'.format(
+    reference_header = '<{addon}/{version}@{site}>'.format(
```

Comment:
@EnTeQuAk This is not a `From` or `To` header. It's used only in `References` and `In-Reply-To` which take message-ids only.